native/packages/react-native-bpk-component-animate-height/src/__snapshots__/BpkAnimateHeight-test.android.js.snap

```diff
@@ -1,138 +1,168 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Android BpkAnimateHeight should render correctly collapsed 1`] = `null`;
```

Comment:
I've regressed the removal of the element from the tree when collapsed. This is on purpose. We prefer to cut down on unnecessary re-renders as opposed to rendering again to remove the component from the tree.